deploifai/cloud_profile/list.py

```diff
@@ -0,0 +1,27 @@
+import click
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated,
+)
+from deploifai.api import DeploifaiAPIError
+
+
+@click.command("list")
+@pass_deploifai_context_obj
+@is_authenticated
+def list_profile(context: DeploifaiContextObj):
+    """
+    Command to list all cloud profiles linked to the current workspace
```

Comment:
I think we can shorten the text a bit more with just `List all cloud profiles in the current workspace`